src/integrationTest/java/uk/gov/hmcts/reform/orgrolemapping/drool/BaseDroolTestIntegration.java

```diff
@@ -171,4 +174,20 @@ protected void triggerCreateOrmMappingApi(UserType userType, List<TestScenario>
             .andReturn();
     }
 
+    @SuppressWarnings({
+        "unchecked",
+        "SameParameterValue" // currently only triggered in Judicial tests
+    })
+    protected Map<String, Boolean> triggerFeatureFlagApi(List<TestScenario> testScenarios) throws Exception {
```

Comment:
We can drop testScenarios from the parameters and MVC Content as it is not needed in the GET request.